providers/vmw_pvrdma/cq.c

```diff
@@ -185,7 +185,7 @@ void pvrdma_cq_clean_int(struct pvrdma_cq *cq, uint32_t qpn)
 				tail = cq->cqe_cnt - 1;
 			curr_cqe = get_cqe(cq, curr);
 			udma_from_device_barrier();
-			if ((curr_cqe->qp & 0xFFFF) != qpn) {
+			if ((curr_cqe->qp & 0xFFFF) != qp_handle) {
```

Comment:
Are you sure that 0xFFFF still valid?

Reply:
The PVRDMA device still does not expose QP handles greater than 0xFFFF, so this is still valid. It's unnecessary though, I can remove it if that is preferred.